launcher/Application.cpp

```diff
@@ -529,7 +533,8 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
         QDir foo(FS::PathCombine(binPath, ".."));
         m_rootPath = foo.absolutePath();
 #elif defined(Q_OS_WIN32)
-        m_rootPath = binPath;
+        QDir foo(FS::PathCombine(QStandardPaths::writableLocation(QStandardPaths::AppDataLocation), ".."));
```

Comment:
Something is not quite right here, if you look a few lines above it says "Root path is used for updates."

Reply:
Does MultiMC auto update?

Reply:
No but it has an updater

Reply:
That's MultiMC's updater which it uses from the launch script which we don't use, so it should be fine.